src/__tests__/e2e/cart.test.js

```diff
@@ -0,0 +1,138 @@
+/**
+ * Copyright © 2018 Elastic Path Software Inc. All rights reserved.
+ *
+ * This is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this license. If not, see
+ *
+ *     https://www.gnu.org/licenses/
+ *
+ *
+ */
+
+/* eslint-disable */
+
+const puppeteer = require('puppeteer');
+
+const host = process.env.TEST_HOST;
+const APP = host || 'http://localhost:8080/';
+const desktopViewport = {
+  width: 1500,
+  height: 700,
+};
+
+describe('Cart feature', () => {
+  
+  test('Change cart line item quantity', async () => {
+    const browser = await puppeteer.launch({
+      args: ['--no-sandbox', '--disable-setuid-sandbox'],
+      slowMo: 10
+    });
+    const page = await browser.newPage();
+    await page.setViewport(desktopViewport);
+    await page.goto(APP);
+
+    const PARENT_CATEGORY_CSS = '.app-header-navigation-component li[data-name="Mens"]';
+    const PRODUCT_CSS = '#category_item_title_link_VESTRI_MENS_SOFT_SHELL_JACKET_RD_LG';
+    const SKU_OPTION_SELECT_CSS = 'div[id="product_display_item_sku_guide"] > div > label[for*="selectorWeight_black"]';
+    const SKU_BUTTON_SELECT_CSS = 'div[id="product_display_item_size_guide"] > div > label[for*="selectorWeight_small"]';
+    const ADD_TO_CART_BUTTON_CSS = 'button[id="product_display_item_add_to_cart_button"]';
+    const QUANTITY_SELECT_CSS = 'input[class="product-display-item-quantity-select form-control form-control-quantity"]';
+    const CART_LINE_ITEM_PRICE_CSS = "div[data-region='itemTotalPriceRegion'] .cart-total-purchase-price";
+
+    const EXPECTED_ITEM_TOTAL = '$340.10';
```

Comment:
This total would be incorrect (along with a few others). Looks like you were pointing to the public Ref Store instance for these tests. We'll need to point to the internal QA instance so the data will match up with the ep-blueprint data we're shipping.